event-management-frontend/src/components/eventCreateOrEdit/ticketsSection/CategoryCard/CategoryCardDumb.tsx

```diff
@@ -0,0 +1,221 @@
+import {
+  Card,
+  CardContent,
+  TextField,
+  FormControl,
+  InputLabel,
+  OutlinedInput,
+  InputAdornment,
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+  FormHelperText,
+  Grid,
+} from "@material-ui/core";
+import { useState } from "react";
+import { useStylesCategoryCard } from "../../../../styles/CategoryCardStyle";
+import React from "react";
+import "../../../../styles/Responsivity.css";
+import { EventCrud } from "../../../../model/EventCrud";
+import { TicketAvailabilityData } from "../../../../model/TicketAvailabilityData";
+import { useTranslation } from "react-i18next";
+import { CategoryCardErrors } from "../../../../model/EventFormErrors";
+
+type Props = {
+  event: EventCrud;
+  id: number;
+  title: string;
+  subtitle: string;
+  price: number;
+  description: string;
+  ticketsPerCategory: number;
+  handleChange: any;
+  ticketData: TicketAvailabilityData[];
+  formErrors: {
+    ticketCategoryDtoList: CategoryCardErrors[];
+  };
+  removeCard: (id: number) => void;
+};
+
+const CategoryCardDumb: React.FC<Props> = (props: Props) => {
+  const [open, setOpen] = useState(false);
+  const [disableMessage, setDisableMessage] = useState("");
+  const [dialogTitle, setDialogTitle] = useState("");
+  const [dialogDescription, setDialogDescription] = useState("");
+
+  const { t } = useTranslation();
```

Comment:
Muta state-urile in componenta Smart. Componenta Dumb este strict prezentationala (nu contine logica)